src/coreclr/src/gc/gc.cpp

```diff
@@ -24989,11 +24989,6 @@ bool gc_heap::frozen_object_p (Object* obj)
 }
 #endif // FEATURE_BASICFREEZE
 
-#ifdef FEATURE_REDHAWK
-// TODO: this was added on RH, we have not done perf runs to see if this is the right
```

Comment:
This is big function. It is unlikely that aggressively inlining it is going to be profitable. I always end up looking at this TODO when investigating these crashes ... decided that it is time to delete it.

Reply:
yeah, it's also hard to say that it won't be profitable. I see it's inlined in 2 places for WKS GC and this is a perf critical function which was probably why someone decided to inline it. can you leave it out of this PR so I can ask if anyone has any context before removing it? I have a bunch of other cleanup type of changes that I wanted to get in soon and can include this unless I can find an argument against it

```
00007ffc`7064ece0 CoreCLR!WKS::gc_heap::relocate_address (unsigned char **)
00007ffc`708a8f40 CoreCLR!SVR::gc_heap::relocate_address (unsigned char **, int)
00007ffc`7064dd80 CoreCLR!WKS::gc_heap::relocate_address =  (inline caller) CoreCLR!WKS::gc_heap::relocate_survivors_in_brick+1c0
00007ffc`7064df80 CoreCLR!WKS::gc_heap::relocate_address =  (inline caller) CoreCLR!WKS::gc_heap::relocate_survivors_in_brick+3c0
```

Reply:
ok